app/frontend/src/components/designer/FormDesigner.vue

```diff
@@ -74,7 +74,7 @@
       </v-card-text>
     </v-card>
 
-    <FormBuilder :form="formSchema" @change="onChangeMethod" :options="{}" />
+    <FormBuilder :form="() => formSchema" @change="onChangeMethod" :options="{}" />
```

Comment:
Since formSchema is now only one-way bound, it needs to be presented as a functional getter; otherwise Vue will warn about it not being a reactive element.